src/medCore/process/medAbstractProcess.h

```diff
@@ -32,6 +32,8 @@ class MEDCORE_EXPORT medAbstractProcess : public dtkAbstractProcess
     medAbstractProcess( medAbstractProcess * parent = NULL );
     virtual ~medAbstractProcess();
 
+    using dtkAbstractProcess::setInput;
```

Comment:
OK but my pb is that I create a process in a plugin. And in a pipeline (which belongs to another plugin), I want to use it.
Here is a clear example of what we are doing right now.

I am in my pipeline (pluginA) and I want to use a thresholding process defined in pluginB. This is how I do it:
```cpp
(pipeline.cpp)

dtkSmartPointer<medAbstractProcess> thresholdProcess = 
      dtkAbstractProcessFactory::instance()->createSmartPointer("itkThresholdingProcess");
thresholdProcess->setInput(maskedMRI);
thresholdProcess->setParameter(thresholds[i], 0); //thresholdValue
...
```
So what is needed is just a medAbstractProcess with accesible *setInput, setParameter*